Library/ViewModels/PledgePaymentMethodsViewModelTests.swift

```diff
@@ -277,24 +300,26 @@ final class PledgePaymentMethodsViewModelTests: TestCase {
       self.reloadPaymentMethodsProjectCountry.assertDidNotEmitValue()
       self.reloadPaymentMethodsSelectedCard.assertDidNotEmitValue()
       self.reloadPaymentMethodsShouldReload.assertDidNotEmitValue()
+      self.reloadPaymentMethodsIsLoading.assertDidNotEmitValue()
 
       self.vm.inputs.configure(with: (User.template, project, Reward.template, .pledge, .discovery))
       self.vm.inputs.viewDidLoad()
 
       self.scheduler.run()
 
       self.reloadPaymentMethodsCards.assertValues([
+        [],
         [
           GraphUserCreditCard.visa,
           GraphUserCreditCard.amex,
           GraphUserCreditCard.discover
         ]
       ])
-      self.reloadPaymentMethodsAvailableCardTypes.assertValues([[true, true, false]])
-      self.reloadPaymentMethodsIsSelected.assertValues([[true, false, false]])
-      self.reloadPaymentMethodsProjectCountry.assertValues([["Brooklyn, NY", "Brooklyn, NY", "Brooklyn, NY"]])
-      self.reloadPaymentMethodsSelectedCard.assertValues([GraphUserCreditCard.visa])
-      self.reloadPaymentMethodsShouldReload.assertValues([true])
+      self.reloadPaymentMethodsAvailableCardTypes.assertValues([[], [true, true, false]])
+      self.reloadPaymentMethodsIsSelected.assertValues([[], [true, false, false]])
+      self.reloadPaymentMethodsProjectCountry.assertValues([[], ["Brooklyn, NY", "Brooklyn, NY", "Brooklyn, NY"]])
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/6baa86e79dff5fcffa0906bf71b0f23e6fc431ec/Library/ViewModels/PledgePaymentMethodsViewModelTests.swift#L320"></span>Line should be 110 characters or less: currently 114 characters<br />
<code>line_length</code> <code>PledgePaymentMethodsViewModelTests.swift:320</code></td>
    </tr>
  </tbody>
</table>